hack/util.sh

```diff
@@ -54,6 +46,22 @@ function setup_env_vars {
 		export CURL_CERT="${MASTER_CONFIG_DIR}/admin.crt"
 		export CURL_KEY="${MASTER_CONFIG_DIR}/admin.key"
 	fi
+}
+
+# setup_temp_dir_vars sets ups all temporary directory variables given the subdirectory name
+function setup_temp_dir_vars() {
```

Comment:
For `test-integration`, for instance, realistically `LOG_DIR`, `ETCD_DATA_DIR`, `SERVER_CONFIG_DIR`, and probably some others are going to be used since we start up etcd and run an apiserver. 

I really would like having the flow of information clearer for our scripts. If we rewrite to follow my proposed style guide, only globals are `ALL_CAPS` but even then it would be nice to not have to dig into utility files to see where they came from. How about we break these into many methods that are more focused?

Also how do you feel about doing a `mkdir -p` in these methods?